news/_posts/2022-02-14-new-theme-announce.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Hello 2022, meet our new Front office theme
+subtitle: Here are some more details about it
+date: 2022-02-14
+authors: [NeOMakinG]
+icon: icon-laptop
+tags: [news, theme, 8.x]
+---
+
+## A new front-office theme is being built 
+
+A new front-office theme is being built, for a long time, many community members have been pushing for a rework of the Classic front-office theme. The maintainer team is happy to announce that a new front-office theme is being built, and you can [find it here](https://github.com/PrestaShop/theme-refacto).
+
+## Why such a decision?
+
+The PrestaShop community provided a [lot of feedback](https://github.com/PrestaShop/PrestaShop/issues/14533) and ideas, with suggestions like:
+
+- Upgrade Bootstrap to version 5, the latest LTS
+- Rework the structure inside  `src`  directory, which was `_dev` before.
+- Remove the `core.js` file that theme developers could not modify or even remove
+- Use [lazy loading](https://en.wikipedia.org/wiki/Lazy_loading).
+- Choose, and use a consistent CSS class naming strategy, for example [BEM](https://github.com/getbem/getbem.com/). This is more maintainable and easier to read.
+- Use [JSON-LD](https://en.wikipedia.org/wiki/JSON-LD) instead of microdata attributes.
+- Add as many hooks as possible to extend the base theme.
```

Comment:
```suggestion
- Add new hooks to help extend the base theme.
```

Reply:
I'm worried that our view of "as many as possible" will still not be enough for some in the community 😅 So this could sound like a false promise